src/transactions/feed/JumpstartTransactionDetailsScreen.tsx

```diff
@@ -90,10 +99,34 @@ function JumpstartTransactionDetailsScreen({ route }: Props) {
   const token = useTokenInfo(transaction.amount.tokenId)
   const isDeposit = transaction.type === TokenTransactionTypeV2.Sent
   const jumpstartContractAddress = transaction.address
+  const networkId = transaction.networkId
+  const tokenAmount = transaction.amount
 
   const walletAddress = useSelector(walletAddressSelector)
   const bottomSheetRef = useRef<BottomSheetRefType>(null)
 
+  const [error, setError] = React.useState<Error | null>(null)
+  const feeCurrencies = useSelector((state) => feeCurrenciesSelector(state, networkId))
+
+  const reclaimStatus = useSelector(jumpstartReclaimStatusSelector)
+  const [isReclaimIniciated, setReclaimInitiated] = useState(false)
```

Comment:
i don't think you need this state variable at all. when you dispatch the jumpstartReclaimStarted action, reclaimStatus will change to 'loading' so you can use that directly

Reply:
I'm using this variable to avoid executing the error state (or even the success state) when the view is loaded the first time and the `reclaimStatus` is from the last interaction.
Is there a way I can execute `dispatch(jumpstartReclaimFlowStarted())` once before showing the component? That would avoid this variable.

Reply:
i had a similar problem with the jumpstart send flow, you can decide when the start or end of the reclaim flow is and reset the state there. you can either reset the state before you first come into this screen, or reset on unmount?